### PR TITLE
install: Update PyPI description

### PR DESCRIPTION
**Pull request by georgebisbas**

### CODE REVIEW

**Review by webmiche — APPROVED** [review]
LGTM.

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import versioneer
-
 from setuptools import find_packages, setup
+from pathlib import Path
+
+# Add README.md as long description
+this_directory = Path(__file__).parent
```
Comment (math-fehr) [inline setup.py:6]: Is there a reason why we are using `Path(__file__)` here instead of `open("README.md")` like in the next few lines?

**Review by math-fehr — APPROVED** [review]
Looks good! I just made a small comment

File: setup.py (hunk 1 above)
Comment (georgebisbas) [reply setup.py:6]: Just copying the docs from here to be safe: 
https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/

**Pull request merged**